app/src/main/java/d/d/meshenger/CallActivity.kt

```diff
@@ -29,7 +29,7 @@ import java.io.IOException
 
 class CallActivity : BaseActivity(), ServiceConnection, SensorEventListener {
     private val buttonAnimationDuration: Long = 400
-    private val CAMERA_PERMISSION_REQUEST_CODE = 2
+    private val CameraPermissionRequestCode = 2
```

Comment:
This is variable needs to be constant and needs to go to a companion object.
No renaming required for this one.